distribution/docker/src/docker/Dockerfile

```diff
@@ -40,11 +40,13 @@ RUN chmod 0660 config/elasticsearch.yml config/log4j2.properties
 #
 # The tini GitHub page gives instructions for verifying the binary using
 # gpg, but the keyservers are slow to return the key and this can fail the
-# build. Instead, we check the binary against a checksum that we have
-# computed.
-ADD https://github.com/krallin/tini/releases/download/v0.18.0/tini${tini_suffix} /tini
-COPY config/tini${tini_suffix}.sha512 /tini.sha512
-RUN sha512sum -c /tini.sha512 && chmod +x /tini
+# build. Instead, we check the binary against a checksum that they provide.
+RUN wget --no-check-certificate --no-cookies --quiet https://github.com/krallin/tini/releases/download/v0.19.0/tini-\$(dpkg --print-architecture) \
```

Comment:
I don't feel comfortable with not checking the certificate here. Is there any particular reason for using this flag?

Reply:
Nope! Let me remove that flag. I think I was looking at some of my flags from an other project and got a little "copy" happy. 👍